io/common/virtual_bind_unbind.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2020 IBM
+# Author: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>
+# Author: Bismruti Bidhibrata Pattjoshi<bbidhibr@in.ibm.com>
+# Author: Abdul Haleem <abdhalee@linux.vnet.ibm.com>
+
+import os
+import time
+import netifaces
+from avocado.utils import genio
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils.process import CmdError
+from avocado.utils.network.interfaces import NetworkInterface
+from avocado.utils.network.hosts import LocalHost
+
+
+class VirtualizationDriverBindTest(Test):
+
+    """
+    virtualized devices can be bound and unbound to drivers.
+    This test verifies that for a given virtualized device.
+    :param device: Name of the virtualized device
+    """
+
+    def setUp(self):
+        """
+        Identify the virtualized device.
+        """
+        smm = SoftwareManager()
+        for pkg in ["net-tools"]:
```

Comment:
@smruti77  is this package name common to all distros ?

Reply:
previously also it was common for all distro  i kept like this.